deployment/permissions/partner.0.ini

```diff
@@ -1345,3 +1345,8 @@ permission207.partnerId = 0
 permission207.type = 1
 permission207.name = APP_TOKEN_UPDATE
 permission207.friendlyName = "Modify application token"
+
```

Comment:
If you add this permission to partner zero, it will be enabled to all partners, please make it depends on your plugin permission.